pkg/kubelet/kubelet.go

```diff
@@ -1477,6 +1479,38 @@ func (kl *Kubelet) updateRuntimeUp() {
 	}
 }
 
+func (kl *Kubelet) reconcileCbr0Loop() {
+	if err := kl.reconcileCbr0(); err != nil {
+		glog.Errorf("Could not reconcile cbr0: %v", err)
+	}
+	for {
+		select {
+		case <-time.After(10 * time.Second):
+			if err := kl.reconcileCbr0(); err != nil {
+				glog.Errorf("Could not reconcile cbr0: %v", err)
+			}
+		}
+	}
+}
+
+func (kl *Kubelet) reconcileCbr0() error {
+	currentNode, err := kl.kubeClient.Nodes().Get(kl.hostname)
```

Comment:
We already do a Nodes().Get() in tryUpdateNodeStatus. What about hanging this function of of there?